src/parser/parser.cpp

```diff
@@ -50,11 +50,22 @@ void Parser::DoNextStep() {
 }
 
 void Parser::ParseNextStep() {
+  // if there are parsing table errors do not perform parsing
+  if (!parsing_table_errors_.empty()) {
+    if (!parser_errors_.empty()) {
+      return;
+    }
+    parser_errors_ = parsing_table_errors_;
+    return;
+  }
   std::string top_symbol = stack_.top();
   std::string current_token = current_string_[current_step_];
   ParsingTable::Table table = table_.GetTable();
   // skip tokens until it is in the first or is a synch token
   while (!IsComplete() && table[top_symbol][current_token] == std::string(ERROR_TOKEN)) {
+    std::string ret;
+    ret += "Parsing error at symbol : " + current_token + " \n";
```

Comment:
According to convention should be "parser: error at symbol: " + current_token + "\n";

Reply:
Yeah okay!